webpack.config.js

```diff
@@ -35,7 +35,8 @@ const config = module.exports = {
     ]
   },
   entry: {
-    'metamaps.bundle': './frontend/src/index.js'
+    'metamaps.bundle': './frontend/src/index.js',
+    'homepage': './frontend/src/homepage.js',
```

Comment:
i'm still uneasy about this - i think it'll be better for us to use one bundle.

There's a webpack feature called code splitting that handles this setup much more gracefully than having multiple entry points/bundles. It's harder to set up but I think it's better long term.

I guess I'm OK leaving this as is for now, but we have to be really vigilant about this or it could cause weird bugs if a library is loaded twice (including a library that's a dependency of other libraries, so we can't just check by looking at package.json).